### PR TITLE
Add role for the old IDS still used by CLF

### PR DESCRIPTION
**Pull request by ajkyffin**

### CODE REVIEW

File: roles/clf-ids/tasks/main.yml [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+
+- name: "Install ImageMagick"
+  package: name=ImageMagick state=present
```
Comment (louise-davies) [inline roles/clf-ids/tasks/main.yml:4]: `state=latest` will ensure this package is up to date

File: roles/clf-ids/defaults/main.yml [hunk 2]
```diff
@@ -0,0 +1,8 @@
+---
+
+clf_ids_version: "4.8.0"
+clf_ids_basedir: "/home/{{ payara_user }}/data/ids/idsbase"
+clf_ids_backupdir: "/home/{{ payara_user }}/data/ids/idsbkup"
+clf_ids_anon_username: "guest"
+clf_ids_anon_password: "guess"
```
Comment (louise-davies) [inline roles/clf-ids/defaults/main.yml:7]: By default doesn't the template use the simple authenticator? In which case these should be the authn.simple credential variables? Or should the template default not be the simple authenticator?

File: roles/clf-ids/tasks/main.yml (hunk 1 above)
Comment (ajkyffin) [reply roles/clf-ids/tasks/main.yml]: fixed https://github.com/icatproject-contrib/icat-ansible/pull/41/commits/d3fd9b9940f9bbce486e2b2b7e3388825a663d53

**Comment by MRichards99** [comment]
Closing on the advice of @ajkyffin

**Pull request closed without merging**